source/index.ts

```diff
@@ -246,22 +241,22 @@ namespace is { // tslint:disable-line:no-namespace
 
 	export const nodeStream = (value: any) => !nullOrUndefined(value) && isObject(value) && function_(value.pipe);
```

Comment:
`plainObject` and `nodeStream` definitely!
`domElement` is a little tricky since it would be `object & { nodeType: 1, nodeName: string, etc.. }`

Reply:
> domElement is a little tricky since it would be object & { nodeType: 1, nodeName: string, etc.. }

But even just `object` (without the `nodeType` etc) for `domElement` would be better than `any`, right?